themes/classic/templates/checkout/_partials/order-confirmation-table.tpl

```diff
@@ -39,7 +39,7 @@
         <div class="order-line row">
           <div class="col-sm-2 col-xs-3">
             <span class="image">
-              {if $product.cover}
+              {if !empty($product.cover)}
```

Comment:
This change seems out of the scope of this PR 🤔 ?

Reply:
This was added by Pierre at the time there was still a bug, it caused the front to crash I believe
Since the bugs in BO have been fixed it shouldn't happen any more but it's an additional safety